### DISCUSSION TITLE
自定义页面部分内容

### DISCUSSION THREAD

**Discussion by cyx1322** [lobehub/lobehub#1279] [Development Question] [answered]
大家新年快乐！
请教两个关于自定义模型的问题：
1.
<img width="268" alt="image" src="https://github.com/lobehub/lobe-chat/assets/81369393/b5fc27a1-f872-4896-9dad-a655b99ff562">
lobechat新版本更新了模型选择的内容，如何在代码中修改，以添加新的模型；如何修改已有的模型，例如修改token数限制，和function call支持情况？
2. 我的API不支持function call，请问如何修改页面元素，隐藏选择插件的按钮？
谢谢！祝大佬们新年快乐！

**Comment by arvinxx** [comment, marked as answer]
这里 https://github.com/lobehub/lobe-chat/blob/main/src/config/modelProviders/openai.ts

**Comment by cyx1322** [comment]
好的，感谢大佬帮助
